Bottom: 3ad0289a97cb8339b00c2faf0e5131d622b04bd0
Top:    4f2e8ed5522720e58806d51ee5032cd803b93d72
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-30 20:48:11 +0100

Fix outline drawing to only apply on copper layers

This matches comments in the code, and fixes (removes) outline drawing
on the solder mask prints.

This partially reverts commit 294494e56695251e63c0da9f243d6f59447f1c9f

"Fix PS rect/outline logic"


---

diff --git a/src/hid/ps/ps.c b/src/hid/ps/ps.c
index 7f2ae8b..1f25448 100644
--- a/src/hid/ps/ps.c
+++ b/src/hid/ps/ps.c
@@ -1001,10 +1001,11 @@ ps_set_layer (const char *name, int group, int empty)
      we want to "print outlines", and we have an outline layer,
      print the outline layer on this layer also.  */
   if (global.outline &&
+      global.is_copper &&
       global.outline_layer != NULL &&
       global.outline_layer != PCB->Data->Layer+idx &&
-      strcmp (name, "outline") != 0 &&
-      strcmp (name, "route") != 0
+      strcmp (name, "outline") &&
+      strcmp (name, "route")
       )
     {
       dapi->draw_layer (global.outline_layer, &global.region, NULL);
